pkg/common/const.go

```diff
@@ -0,0 +1,7 @@
+package common
+
+const (
+	TektonOpMinSupportedVersion = "v0.50.0"
+	TektonOpMinSupportedMajor   = 0
+	TektonOpMinSupportedMinor   = 50
```

Comment:
For future - we can use the [semver](https://pkg.go.dev/golang.org/x/mod/semver) package to extract major/minor versions